samples/vsphere/sso/README.md

```diff
@@ -11,3 +11,17 @@ $ python external_psc_sso_workflow.py --lsurl https://<server>/lookupservice/sdk
 ```
 * Testbed Requirement:
     - 1 vCenter Server
+
+### Deprecation Notice
+Starting vCenter server release 7.0, External Platform Services Controller (PSC) is no longer supported. All PSC services are consolidated into vCenter Server.
+https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenter.configuration.doc/GUID-135F2607-DA51-47A5-BB7A-56AD141113D4.html
+In view of the above, related samples (ex: external_psc_sso_workflow) and other related files are deprecated and will be removed in next major SDK release.
+
+Consequently, lookupservice WSDL files will also be removed in next major SDK release. Use well known URL path (https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-apis-sdks-introduction/GUID-B625C8FE-5E15-4918-98C0-69313E5880FB.html) instead of lookupservice.
+
+For SSO, service endpoint is: "https://{domain}/STS/STSService"
```

Comment:
We found out that this URL is case sensitive. Correct URL is "https://{domain}/sts/STSService"
Please update.

Reply:
Done